### ISSUE TITLE
Toast Notification Accessibility Guidance

### ISSUE THREAD

**Issue by jcfilben** [grommet/hpe-design-system#2781]
The Design System's [Toast Notification accessibility guidance](https://design-system.hpe.design/templates/toast-notifications?q=toast#accessibility) says `Toast notifications should be read aloud when they appear on a user's screen. To accomplish this, toasts are given the role of "log" with default aria-live set to "polite".` However when toasts are given `role="log"` and `aria-live="polite"` they are not read out by the screen reader. See [codesandbox](https://codesandbox.io/s/grommet-v2-template-forked-dcm4ly?file=/index.js) example. We should update the guidance on this and evaluate if we should change the grommet code or if it is up to the user to determine how they want their toast notification read out with aria-live ('assertive' vs 'polite').
